build.assets/Dockerfile-centos7

```
@@ -92,6 +92,33 @@ RUN mkdir -p /opt && cd /opt && \
     cd /opt/libbpf-${LIBBPF_VERSION}/src && \
     scl enable devtoolset-11 "make && BUILD_STATIC_ONLY=y DESTDIR=/opt/libbpf make install"
 
+# Install newer libbpf.
+FROM centos:7 AS libbpf-1-0-1
+
+# Install required dependencies.
+RUN yum groupinstall -y 'Development Tools' && \
+    yum install -y epel-release && \
+    yum update -y && \
+    yum -y install centos-release-scl-rh && \
+    yum install -y \
+    # required by libbpf
+    centos-release-scl \
+    # required by libbpf
+    devtoolset-11-gcc* \
+    # required by libbpf
+    devtoolset-11-make \
+    # required by libbpf
+    elfutils-libelf-devel-static \
+    git \
+    # required by libbpf
+    scl-utils \
+    yum clean all
```

```suggestion
    yum install -y \
        git \
        # required by libbpf
        centos-release-scl \
        devtoolset-11-gcc* \
        devtoolset-11-make \
        elfutils-libelf-devel-static \
        scl-utils \
    yum clean all
```

Although we are in a BPF-specific step, so we could do away with all the comments too.